tests/data/utils.py

```diff
@@ -24,7 +24,9 @@ def iterate_over_rules():
         leaf_dir = os.path.basename(dir_name)
         rel_dir = '.' + dir_name[len(_DIR):]
         if leaf_dir.startswith('rule_'):
-            result = Rule(rel_dir, _SSG_PREFIX + leaf_dir, files)
+            # filter out swap files
+            scripts = filter(lambda x: not x.endswith("swp"), files)
```

Comment:
I would like ask to exclude vim backup files as well, their names ends with `~`.

Reply:
Do we want to just whitelist `.sh` scripts? I think that'd make more sense to me personally, unless the few files not ending in `.sh` (like `group_system/group_auditing/group_auditd_configure_rules/group_audit_privileged_commands/rule_audit_rules_privileged_commands/rhel7_privileged.rules`) need to be executed, but still, I'd rather have a whitelist rather than a blacklist; seems less fragile.

Reply:
aha, ok

Reply:
I also strongly prefer whitelist

Reply:
aha